mnc/xengine_beamformer_control.py

```diff
@@ -207,17 +207,17 @@ def set_beam_vlbi_dest(self, addr='10.41.0.73', port=21001):
             with AllowedPipelineFailure(p):
                 p.beamform_vlbi_output.set_destination(addr, port)
                 
-    def _freq_to_pipeline(self, freq_to_find):
+    def _freq_range_to_pipeline(self, first_freq, last_freq):
         """
-        Given a frequency in Hz, return the pipeline index where that frequency
-        can be found in the data.  Raises a ValueError if there does not appear
-        to be a corresponding pipeline.
+        Given the first and last values of a frequency range in Hz, return the
+        pipeline index where that frequency can be found in the data.  Raises a
+        ValueError if there does not appear to be a corresponding pipeline.
         """
         
         for i,freq in enumerate(self.freqs):
-            if freq_to_find >= freq[0] and freq_to_find <= freq[-1]:
+            if first_freq == freq[0] and last_freq == freq[-1]:
                 return i
-        raise ValueError(f"Cannot associate {freq_to_find/1e6:.3f} MHz with any pipeline currently under control")
+        raise ValueError(f"Cannot associate {first_freq/1e6:.3f} to {last_freq/1e6:.3f} MHz with any pipeline currently under control")
```

Comment:
log the value error? The catch -> pass block down below means that no one will see this message, right?